pvlib/tests/iotools/test_pvgis.py

```diff
@@ -20,23 +20,22 @@
 testfile_radiation_csv = DATA_DIR / \
     'pvgis_hourly_Timeseries_45.000_8.000_SA_30deg_0deg_2016_2016.csv'
 testfile_pv_json = DATA_DIR / \
-    'pvgis_hourly_Timeseries_45.000_8.000_CM_10kWp_CIS_5_2a_2013_2014.json'
+    'pvgis_hourly_Timeseries_45.000_8.000_SA2_10kWp_CIS_5_2a_2013_2014.json'
 
 index_radiation_csv = \
     pd.date_range('20160101 00:10', freq='1h', periods=14, tz='UTC')
 index_pv_json = \
-    pd.date_range('2013-01-01 00:55', freq='1h', periods=10, tz='UTC')
+    pd.date_range('2013-01-01 00:10', freq='1h', periods=10, tz='UTC')
 
 columns_radiation_csv = [
     'Gb(i)', 'Gd(i)', 'Gr(i)', 'H_sun', 'T2m', 'WS10m', 'Int']
 columns_radiation_csv_mapped = [
     'poa_direct', 'poa_sky_diffuse', 'poa_ground_diffuse', 'solar_elevation',
     'temp_air', 'wind_speed', 'Int']
 columns_pv_json = [
-    'P', 'Gb(i)', 'Gd(i)', 'Gr(i)', 'H_sun', 'T2m', 'WS10m', 'Int']
+    'P', 'G(i)', 'H_sun', 'T2m', 'WS10m', 'Int']
```

Comment:
How come the in-plane components got dropped here?  It seems like the API still returns them -- compare this from the test file:
```json
{"time": "20130101:0010", "P": 0.0, "G(i)": 0.0, "H_sun": 0.0, "T2m": -0.97, "WS10m": 1.52, "Int": 0.0}
```
versus this from the API itself (using the `v5_2` url):
```json
{"time": "20130101:0010", "P": 0.0, "Gb(i)": 0.0, "Gd(i)": 0.0, "Gr(i)": 0.0, "H_sun": 0.0, "T2m": -0.97, "WS10m": 1.52, "Int": 0.0}
```

Reply:
I had generated the file with `components=0` but not updated the test instances to match. I guess this is the flipside of mock testing... 

Since the affected tests were only mock test, there is no real connection with what is actually being requested and the passed test file.